app/src/main/java/io/gnosis/safe/ui/settings/view/NamedAddressItem.kt

```diff
@@ -86,4 +89,19 @@ class NamedAddressItem @JvmOverloads constructor(
     private fun applyAttributes(context: Context, a: TypedArray) {
         binding.namedAddressItemSeparator.visible(a.getBoolean(R.styleable.NamedAddressItem_show_named_address_separator, false))
     }
+
+    override fun onPrepareLoad(placeHolderDrawable: Drawable?) {}
+
+    override fun onBitmapFailed(e: Exception?, errorDrawable: Drawable?) {}
+
+    override fun onBitmapLoaded(bitmap: Bitmap?, from: Picasso.LoadedFrom?) {
+        with(binding) {
+            blockies.setAddress(null)
```

Comment:
we could override setImageBitmap in KnownAddressImageView and set blockies to null there

Reply:
Is this still applicable?